public/app/percona/backup/components/ScheduledBackups/ScheduledBackups.tsx

```diff
@@ -231,8 +240,8 @@ export const ScheduledBackups: FC = () => {
   const featureSelector = useCallback(getPerconaSettingFlag('backupEnabled'), []);
 
   useEffect(() => {
-    getData();
     dispatch(fetchStorageLocations());
+    getData();
```

Comment:
Why did we change the order here? Does this change something?

Reply:
Yes if fetching storage locations is after getData it will fetch locations after data loads and it blinks for a moment

Reply:
Yeah, you are right. I moved the fetch storage location to the correct position so It is in sync with the data.